phpcs.ruleset.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Plugins">
+	<description>Generally-applicable sniffs for WordPress plugins</description>
+
+	<rule ref="WordPress-Core">
+		<exclude name="Squiz.Commenting.FunctionComment.ParamCommentFullStop" />
+		<exclude name="Squiz.Commenting.InlineComment.InvalidEndChar" />
+		<exclude name="Generic.PHP.NoSilencedErrors.Discouraged" />
+		<exclude name="WordPress.CSRF.NonceVerification.NoNonceVerification" />
```

Comment:
Completely agree. My thinking was to make less-invasive style changes, and then to make targeted, more invasive changes in a subsequent pass.